packages/node_modules/@webex/plugin-meetings/src/locus-info/selfUtils.js

```diff
@@ -51,13 +56,39 @@ SelfUtils.getSelves = (oldSelf, newSelf, deviceId) => {
   updates.moderatorChanged = SelfUtils.moderatorChanged(previous, current);
   updates.isMediaInactiveOrReleased = SelfUtils.wasMediaInactiveOrReleased(previous, current);
 
+  updates.isMediaInactive = SelfUtils.isMediaInactive(previous, current);
+
   return {
     previous,
     current,
     updates
   };
 };
 
+SelfUtils.isMediaInactive = (previous, current) => {
+  if (previous && previous.joinedWith && previous.joinedWith.mediaSessions &&
+     current && current.joinedWith && current.joinedWith.mediaSessions) {
+    const previousMediaStatus = SelfUtils.getMediaStatus(previous.joinedWith.mediaSessions);
+    const currentMediaStatus = SelfUtils.getMediaStatus(current.joinedWith.mediaSessions);
+
+    if (previousMediaStatus.audio.state !== MEDIA_STATE.inactive && currentMediaStatus.audio.state === MEDIA_STATE.inactive && currentMediaStatus.audio.direction !== MEDIA_STATE.inactive) {
```

Comment:
Sorry, one more question about this check. Why would direction not being inactive mean media is inactive? Does `getMediaStatus` not update direction?

`currentMediaStatus.audio.direction !== MEDIA_STATE.inactive`

Reply:
If someone had not request for media like video to be sent and received intentionally. Then we should not reconnect as its a valid scenario 

So the direction and status is the one we comparer if its sendOnly , recvonly and sendRecv but actually the media was inactive then we trigger reconenct